bin/jobsub_

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3 -I
```

Comment:
Rather than have a separate executable to get the version, would it be possible to just add these flags (lines 46-57 here) into the lib/get_parser.py file so all jobsub_* executables inherit it?  

If so, I think it should be in the base parser:
https://github.com/marcmengel/jobsub_lite/blob/master/lib/get_parser.py#L102

What do you think?